src/arena.c

```diff
@@ -55,6 +55,11 @@ static unsigned nbins_total;
 
 static unsigned huge_arena_ind;
 
+const arena_config_t arena_config_default = {
+	.extent_hooks = (extent_hooks_t *)&ehooks_default_extent_hooks,
```

Comment:
Nit: we can't use designated initializers because they don't compile on some older versions of MSVC that we still try to support. (Unfortunately, this isn't well tested in CI).